tests/python/rest_api/test_tasks.py

```diff
@@ -1723,6 +1745,91 @@ def test_create_task_with_cloud_storage_and_check_data_sorting(
             for image_name, frame in zip(filenames, data_meta.frames):
                 assert frame.name.rsplit("/", maxsplit=1)[1] == image_name
 
+    @pytest.mark.with_external_services
+    @pytest.mark.parametrize(
+        "cloud_storage_id, org",
+        [
+            (1, ""),
+        ],
+    )
+    def test_create_task_with_cloud_storage_and_check_retrieve_data_meta(
+        self,
+        cloud_storage_id: int,
+        org: str,
+        cloud_storages,
+        request,
+    ):
+        cloud_storage = cloud_storages[cloud_storage_id]
+
+        data_spec = {
+            "start_frame": 2,
+            "stop_frame": 6,
+            "frame_filter": "step=2",
+        }
+
+        task_id, _ = self._create_task_with_cloud_data(
+            request=request,
+            cloud_storage=cloud_storage,
+            use_manifest=False,
+            use_cache=False,
+            server_files=["test/video/video.avi"],
+            org=org,
+            spec=data_spec,
+            data_type="video",
+        )
+
+        with make_api_client(self._USERNAME) as api_client:
+            data_meta, _ = api_client.tasks_api.retrieve_data_meta(task_id)
+
+        assert data_meta.start_frame == 2
+        assert data_meta.stop_frame == 6
+        assert data_meta.size == 3
+
+    @pytest.mark.with_external_services
+    @pytest.mark.parametrize(
+        "cloud_storage_id, org",
+        [
+            (1, ""),
+        ],
+    )
+    def test_create_task_with_cloud_storage_using_advance_params_and_large_dataset(
+        self,
+        cloud_storage_id: int,
+        org: str,
+        cloud_storages,
+        request,
+    ):
+        cloud_storage = cloud_storages[cloud_storage_id]
+
+        data_spec = {
+            "start_frame": 10,
+            "stop_frame": 30,
+            "frame_filter": "step=10",
+        }
+        process = psutil.Process()
+        before_memory = process.memory_info().rss
+        task_id, _ = self._create_task_with_cloud_data(
+            request=request,
+            cloud_storage=cloud_storage,
+            use_manifest=False,
+            use_cache=False,
+            server_files=["test/video/video.avi"],
+            org=org,
+            spec=data_spec,
+            data_type="video",
+            video_frame_count=4000,
+        )
+        after_memory = process.memory_info().rss
+        print(before_memory, after_memory)
+        print(f"Memory usage: {after_memory - before_memory}")
```

Comment:
I doubt the `print` calls check anything. Also, rss is not maximum memory used, so has no relation to the actual memory requirements of the operation being tested.

Reply:
Yeah, I forgot to remove the `prints`, fixed that. And changed the memory usage calculation.